### DISCUSSION TITLE
Lame mp3 control bits

### DISCUSSION THREAD

**Discussion by stillusingmp3** [enzo1982/freac#475] [Audio Format Discussion]
Not too sure what control bits do in the lame mp3 settings.

![Screenshot_1](https://user-images.githubusercontent.com/130705373/231880274-b305e500-4962-40dd-bc78-99fa4f2a5161.jpg)

**Comment by enzo1982** [comment]
These are just flags set in the MP3 frame headers. Some players can read these flags and display their values.

These flags have no influence on audio quality or file size.

**Reply by stillusingmp3** [reply]
ok thank you
